src/PortingAssistant.Client.Client/Reports/ReportExporter.cs

```diff
@@ -66,9 +67,8 @@ public ReportExporter(ILogger<ReportExporter> logger)
                     Directory.CreateDirectory(FileDir);
                     List<PackageAnalysisResult> packageAnalysisResults = new List<PackageAnalysisResult>();
                     Dictionary<PackageVersionPair, string> packageAnalysisResultErrors = new Dictionary<PackageVersionPair, string>();
-
-                    projectAnalysResult.PackageAnalysisResults.ToList()
-                    .ForEach(p =>
+                    var sorted = projectAnalysResult.PackageAnalysisResults.OrderBy(c=>c.Key.PackageId).ToList();
```

Comment:
There was a performance regression identified- do we need to sort here?

Reply:
i need to compare the result for packages in order.  sorted will make it easier.  but can revert it back